src/openzaak/components/zaken/models/zaken.py

```diff
@@ -537,7 +537,11 @@ class RelevanteZaakRelatie(models.Model):
     url = FkOrServiceUrlField(
         fk_field="_relevant_zaak",
         url_field="_relevant_zaak_url",
-        help_text=_("URL-referentie naar de ZAAK."),
+        help_text=_(
+            "URL-referentie naar de ZAAK. "
+            "(Het ZAAKTYPE van deze ZAAK hoeft niet te zijn toegevoegd "
+            "aan de gerelateerdezaaktypen/zaaktypenrelaties van het andere ZAAKTYPE.)"
```

Comment:
Let's stick to the same casing as in the API spec for properties
```suggestion
            "aan de gerelateerdeZaaktypen van het andere ZAAKTYPE.)"
```